samples/scala-value-entity-customer-registry/src/main/scala/customer/action/customer_action/CustomerActionImpl.scala

```diff
@@ -0,0 +1,19 @@
+/* This code was generated by Akka Serverless tooling.
+ * As long as this file exists it will not be re-generated.
+ * You are free to make changes to this file.
+ */
+package customer.action.customer_action
+
+import com.akkaserverless.scalasdk.action.Action
+import com.akkaserverless.scalasdk.action.ActionCreationContext
+import com.google.protobuf.empty.Empty
+import customer.api.customer_api.Customer
+
+/** An action. */
+class CustomerActionImpl(creationContext: ActionCreationContext) extends AbstractCustomerAction {
+
+  /** Handler for "Create". */
+  override def create(customer: Customer): Action.Effect[Empty] = {
+    throw new RuntimeException("The command handler for `Create` is not implemented, yet")
```

Comment:
Missing the forward in the Java one

Reply:
Ah right thanks, must still implement the action

Reply:
added